package.json

```diff
@@ -16,18 +16,26 @@
     "popsicle": "^9.2.0"
   },
   "devDependencies": {
-    "eslint": "^4.17.0"
+    "coveralls": "^3.0.0",
+    "eslint": "^4.17.0",
+    "faucet": "0.0.1",
```

Comment:
set this to `^0.0.1` (semver range)?

Reply:
Huh. This is how `npm install` added it. I'll fix it.

Reply:
> Huh. This is how npm install added it. I'll fix it.

I have $1 internet buck saying your **~/.npmrc** file has `save-exact=true`, which would drop any semver ranges.

Reply:
You owe me an internet buck ...

```
groovetop:blurts-server lcrouch$ cat ~/.npmrc
//registry.npmjs.org/:_authToken = [redacted]
groovetop:blurts-server lcrouch$
```

Reply:
Double or nothing on `$ npm config list`?
Or possibly `npm config ls -l | grep save` may show if you've overridden your `save-prefix` from `^`.